Makefile

```diff
@@ -22,9 +22,14 @@ docker-build:
 		--build-arg "OPENTUTOR_CLIENT_VERSION=$(OPENTUTOR_CLIENT_VERSION)" \
 	.
 
-.PHONY: format
-format: node_modules/prettier
-	npm run format
+PHONY: format
+format: LICENSE LICENSE_HEADER node_modules/license-check-and-add
+	npm ci && npm run license:fix
```

Comment:
small thing, but better to have `license` be it's own make rule, so:

```Makefile
.PHONY: license
license:  LICENSE LICENSE_HEADER  node_modules/license-check-and-add
	npm run license:fix

.PHONY: format
format:
	$(MAKE) license
```

...also don't do the `npm ci` in the license rule, the dependency on `node_modules/license-check-and-add` is there specifically to handle that